### PR TITLE
No info on environment variables in the src folder (#33110)

### PR DESCRIPTION
**Pull request by atenie**
Proposes fix to #33110

Adds info on environment variables and the /src folder in the following docs:

- https://nextjs.org/docs/advanced-features/src-directory
- https://nextjs.org/docs/basic-features/environment-variables#loading-environment-variables

## Bug

- [ ] Related issues linked using `fixes #number`
- [ ] Integration tests added
- [ ] Errors have helpful link attached, see `contributing.md`

## Feature

- [ ] Implements an existing feature request or RFC. Make sure the feature request has been accepted for implementation before opening a PR.
- [ ] Related issues linked using `fixes #number`
- [ ] Integration tests added
- [x] Documentation added
- [ ] Telemetry added. In case of a feature if it's used or not.
- [ ] Errors have helpful link attached, see `contributing.md`

## Documentation / Examples

- [x] Make sure the linting passes by running `yarn lint`

### CODE REVIEW

**Review by timneutkens — DISMISSED** [review]

File: docs/basic-features/environment-variables.md [hunk 1]
```diff
@@ -76,6 +76,9 @@ export async function getStaticProps() {
 > CORRECT=pre\$A
 > ```
 
+> **Note**: If you are using a `/src` folder, please note that Next.JS will load the .env files **only** from the parent
```
Comment (timneutkens) [inline docs/basic-features/environment-variables.md:79]:
```suggestion
> **Note**: If you are using a `/src` folder, please note that Next.js will load the .env files **only** from the parent
```

File: docs/basic-features/environment-variables.md [hunk 2]
```diff
@@ -76,6 +76,9 @@ export async function getStaticProps() {
...
+> folder and **not** from the `/src` folder.
```
Comment (timneutkens) [inline docs/basic-features/environment-variables.md:80]:
```suggestion
```

File: docs/basic-features/environment-variables.md (hunk 1 above)
Comment (timneutkens) [inline docs/basic-features/environment-variables.md]:
```suggestion
> **Note**: If you are using a `/src` folder, please note that Next.JS will load the .env files **only** from the parent folder and **not** from the `/src` folder.
```

File: docs/basic-features/environment-variables.md (hunk 2 above)
Comment (atenie) [reply docs/basic-features/environment-variables.md:80]: Let's keep the word "folder".

File: docs/basic-features/environment-variables.md [hunk 3]
```diff
@@ -76,6 +76,9 @@ export async function getStaticProps() {
 > CORRECT=pre\$A
 > ```
 
+> **Note**: If you are using a `/src` folder, please note that Next.js will load the .env files **only** from the parent
```
Comment (timneutkens) [inline docs/basic-features/environment-variables.md:79]:
```suggestion
> **Note**: If you are using a `/src` folder, please note that Next.js will load the .env files **only** from the parent folder and **not** from the `/src` folder.
```

**Review by timneutkens — APPROVED** [review]

File: docs/basic-features/environment-variables.md (hunk 2 above)
Comment (timneutkens) [reply docs/basic-features/environment-variables.md]: I moved it up to the line above, unfortunately couldn't do a multi-line change with the review tool.

**Pull request merged**